py/makemoduledefs.py

```diff
@@ -40,6 +41,13 @@ def generate_module_table_header(modules):
     for module_name, obj_module in modules:
         mod_def = "MODULE_DEF_{}".format(module_name.upper())
         mod_defs.add(mod_def)
+        if "," in obj_module:
+            sys.stderr.write(
```

Comment:
To be consistent with error output in other scripts, this should use `print(..., file=sys.stderr)`.